test/external/handle-filter

```diff
@@ -6,15 +6,15 @@ class Filter < Sensu::Handler
     puts "Event: #{@event.inspect}"
   end
 
-  def api_request(*)
+  def api_request(*_args)
```

Comment:
I get that same behaviour with and without the `_` so I don't think that is a problem.